docs/docsite/rst/user_guide/intro_inventory.rst

```diff
@@ -16,8 +16,6 @@ Not only is this inventory configurable, but you can also use multiple inventory
 pull inventory from dynamic or cloud sources or different formats (YAML, ini, etc), as described in :ref:`intro_dynamic_inventory`.
 Introduced in version 2.4, Ansible has inventory plugins to make this flexible and customizable.
 
-The TOML inventory format was introduced in version 2.8.
```

Comment:
Maybe put it somewhere else?

Reply:
This info already lives in the plugins DOCUMENTATION, and will be available when viewing the docs for the module.

For now, we aren't going to promote it here.